cmd/otelcorecol/go.mod

```diff
@@ -59,13 +59,18 @@ require (
 	go.opentelemetry.io/collector/semconv v0.54.0 // indirect
 	go.opentelemetry.io/contrib/instrumentation/google.golang.org/grpc/otelgrpc v0.32.0 // indirect
 	go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp v0.32.0 // indirect
+	go.opentelemetry.io/contrib/propagators/autoprop v0.33.0 // indirect
```

Comment:
IIUC this is a direct dependency, right? So it should be on the other require block

Reply:
Think this is In the context of cmd/otelcorecol module - the files in the module doesn't refer to autoprop directly (just as they don't refer to other indirect dependencies, even though they are direct dependencies for exporters, receivers, service and extensions etc. 

The top level go.mod present in the root folder has go.opentelemetry.io/contrib/propagators/autoprop as direct dependency though.

Reply:
Right :facepalm: looking at the wrong file, thanks!